pocs/utils/error.py

```diff
@@ -1,28 +1,39 @@
 import sys
 
-from astropy.utils.exceptions import AstropyWarning
+from pocs.utils.logger import get_root_logger
 
-from pocs.base import PanBase
+logger = get_root_logger()
 
 
-class PanError(AstropyWarning, PanBase):
+class PanError(Exception):
 
     """ Base class for Panoptes errors """
 
     def __init__(self, msg=None, exit=False):
-        PanBase.__init__(self)
         if msg:
-            if exit:
-                self.exit_program(msg)
-            else:
-                self.logger.error('{}: {}'.format(self.__class__.__name__, msg))
-                self.msg = msg
+            logger.error('{}: {}'.format(self.__class__.__name__, msg))
 
-    def exit_program(self, msg='No reason specified'):
+        if exit:
+            self.exit_program(msg)
+
+        self.msg = msg
```

Comment:
How about setting the msg immediately on entry to `__init__`? Then you can make use of `__str__` to generate the log message.